api/routes/api_routes.py

```diff
@@ -0,0 +1,13 @@
+from flask import Blueprint,request,json
+from api.controller.EducationController import educationcontroller
+
+api = Blueprint("api", __name__)
+@api.route('/education', methods=['GET'])
+def home():
+    return educationcontroller.index()
+# @api.route('/registration', methods=['GET'])
```

Comment:
These routes are incorrect, but they will have updates to convert them to education CRUD in the next PR